src/devices/vdml_imu.c

```diff
@@ -87,8 +87,7 @@ double imu_get_heading(uint8_t port) {
 	claim_port_f(port - 1, E_DEVICE_IMU);
 	ERROR_IMU_STILL_CALIBRATING(port, device, PROS_ERR_F);
 	double rtn = vexDeviceImuDegreesGet(device->device_info) + ((imu_data_s_t*)registry_get_device(port - 1)->pad)->heading_offset;
-	if (rtn > IMU_HEADING_MAX) rtn -= IMU_HEADING_MAX;
-	if (rtn < 0) rtn += IMU_HEADING_MAX;
+	rtn = fmod((rtn + IMU_HEADING_MAX), (double) IMU_HEADING_MAX);
 	return_port(port - 1, rtn);
```

Comment:
```suggestion
	// Restricting value to raw boundaries
	return_port(port - 1, fmod((rtn + IMU_HEADING_MAX), (double) IMU_HEADING_MAX));
```
For readability, I think adding a comment and do return_port should be fine for this.